@types/pal/wasm.d.ts

```diff
@@ -19,6 +19,8 @@ declare module 'pal/wasm' {
      */
     export function fetchBuffer (binaryUrl: string): Promise<ArrayBuffer>;
 
+    export function fetchUrl(binaryUrl: string): Promise<string>;
```

Comment:
Miss API comments for this new function `fetchUrl`. 
And what's its functionality?  The argument `binaryUrl` is already an URL, why need to `fetch` an URL by another URL?

Reply:
What I actually need is the actual path address of this URL, not the binary file. For example, if the passed path is "external:emscripten/webgpu/glslang.wasm", I need to perform an actual conversion so that it can be recognized by all platforms.